skin/adminhtml/default/default/boxes.css

```diff
@@ -220,6 +220,21 @@ tr.dynamic-grid input.input-text { width:154px; }
 .available { color:#080; font-weight:bold; }
 .not-available { color:#800; }
 
+/* Grid - Copyable Cells */
+.icon-copy {
```

Comment:
can't we re-use the icon-copy class that's already in override.css?

Reply:
Not sure how to use the override.css in the openmage theme in the legacy.

Reply:
if it's not in the legacy theme maybe we should migrate it away from override.css, put it in the legacy so that it's available for both and we don't have to duplicate the classes

Reply:
You are right, we should.